### DISCUSSION TITLE
Feature: Show/Hide Persona Bar

### DISCUSSION THREAD

**Discussion by erw13n** [dnnsoftware/Dnn.Platform#5581] [Ideas]
Nice to have if we can Show/Hide persona bar and not load related Javascript & css when persona bar is hidden.
This can slightly increase the web speed when Host/Admin login.

**Comment by valadas** [comment]
I am not sure what you mean, if you are not logged-in then there is no persona bar (and related html/js/css. When you have a role that needs to show the persona bar then it loads what the PB needs but not the modules on it yet, then when you open one module, it has to load that module. Not sure I understand your suggestion here...

**Comment by erw13n** [comment]
When Host/Admin sign-in, sometimes persona bar is not needed:
- Move between pages
- Working on module configuration
- Or just refresh the page

We not always use all persona bar features, mostly only for Editing the page. (For switching to Edit-Mode)
By not load persona bar, it speed up a bit especially with slow internet connection.

**Reply by mitchelsellers** [reply]
My personal $0.02 here is that it is going to be a micro-optimization that will introduce more steps and processes that could break with the persona bar.  I understand the desire, but the relative cost of loading the persona bar is very, very minimal.
